models/job.go

```diff
@@ -62,6 +62,10 @@ const (
 	JobRetryEvent JobEventType = "retry"
 )
 
+var faiulreEvents = []JobEventType{
```

Comment:
we can keep this internal to Should Notify or a method on the struct

Reply:
sure

Reply:
okay